doc/policy/README.md

```diff
@@ -0,0 +1,11 @@
+# Mempool Policy
+
+**Mempool Policy** (aka Transaction Relay Policy) is the node's set of validation rules, in addition
```

Comment:
Not sure it's a good idea to call them "rules". Maybe "conditions"?

Reply:
Is it because "rules" sounds too strict? I think "conditions" is too weak otoh, and I've used it to mean something else in the docs.

I've added a clarification that these are _local_ rules, only for mempool transactions, *not* used for block transactions. Hopefully that makes it clear to the reader